modules/gateway/nodes_test.go

```diff
@@ -77,7 +77,7 @@ func TestRandomNode(t *testing.T) {
 	}
 	for node, count := range nodes {
 		if count == 0 { // 1-in-200000 chance of occuring naturally
```

Comment:
hmmm. Might want an extra 2-3 zeroes.

nbd I guess, but if you have 10 of these then it's down to 1-in-20k, and we run 'test' quite a few times. More than once per commit I imagine.